src/pages/texting.astro

```diff
@@ -8,7 +8,7 @@ import "animate.css";
 	title='Spoke Texting - With the Ranks'
 	bannerCategory='Spoke'
 	bannerTitle='Spoke Texting'
-	bannerSubtitle='Built by veterans of the Bernie 2020 campaign and the movements and organizations surrounding it, Spoke is the lowest cost peer-to-peer texting platform on the market.'
+	bannerSubtitle='Built by veterans of Bernie's presidential campaigns and the movements and organizations that emerged alongside them, Spoke is the lowest cost peer-to-peer texting platform on the market.'
```

Comment:
Might need double quotes or to escape the single quote here, I believe causing the build to fail.

Reply:
ofc lol, thx for the catch, fixing!